internal/rest/router.go

```diff
@@ -41,7 +41,7 @@ func (r *router) addRoutes() {
 
 	r.httpRouter.POST("/transactions", r.sendTransaction)
 	r.httpRouter.GET("/receipts", r.handleReceipts)
-	r.httpRouter.GET("/receipts/:receiptId", r.handleReceipts)
+	r.httpRouter.GET("/receipts/:id", r.handleReceipts)
 
 	// router.POST("/eventstreams", r.createStream)
```

Comment:
Can we remove all these commented out routes? Is it adding any value right now?

Reply:
these are intentionally added to signal future work in case someone is interested in contributing